File: spotifyarch/src/main/java/com/timothy/spotifyarchitecture/retrofit/models/PlaylistSimple.java
package com.timothy.spotifyarchitecture.retrofit.models;

import android.arch.persistence.room.Embedded;
import android.os.Parcel;

/**
 * <a href="https://developer.spotify.com/web-api/object-model/#playlist-object-simplified">Playlist object model (simplified)</a>
 */
public class PlaylistSimple extends PlaylistBase {
    @Embedded
    public PlaylistTracksInformation tracks;

    public PlaylistSimple() {
    }
}

File: spotify-web/src/main/java/com/timmahh/spotifyweb/retrofit/models/PlaylistSimple.kt
package com.timmahh.spotifyweb.retrofit.models

/**
 * [Playlist object model (simplified)](https://developer.spotify.com/web-api/object-model/#playlist-object-simplified)
 */
class PlaylistSimple(val tracks: PlaylistTracksInformation = PlaylistTracksInformation(), collaborative: Boolean, external_urls: Map<String, String>, href: String, id: String, images: List<Image>, name: String, owner: UserPublic, is_public: Boolean, snapshot_id: String, type: String, uri: String) : PlaylistBase(collaborative, external_urls, href, id, images, name, owner, is_public, snapshot_id, type, uri)
